Course.java

```diff
@@ -0,0 +1,47 @@
+public class Course {
+  private String courseName;
+  public int classSize =0;
+  private ArrayList<Person> StudentList;
```

Comment:
:nit: Is the convention to use uppercase or lowercase for ArrayList variable names? Either way, just pick one convention.